docs/userguides/compile.md

```diff
@@ -81,3 +81,31 @@ Alternatively, configure it to always happen:
 compile:
   use_dependencies: true
 ```
+
+## Settings
+
+Generally, configure compiler plugins using your `ape-config.yaml` file.
+For example, when using the `vyper` plugin, you can configure settings under the `vyper` key:
+
+```yaml
+vyper:
+  version: 0.3.10
+```
+
+You can also configure adhoc settings in Python code:
+
+```python
+from ape import compilers
+
+settings = {"vyper": {"version": "0.3.7"}, "solidity": {"version": "0.8.0"}}
+compilers.compile(
```

Comment:
when I try this I get the error: 
`AttributeError: 'str' object has no attribute 'suffix'`

Reply:
- [ ] allow str paths to work for simpler API in this case use-case (avoids needing to wrap everything in Path)

Reply:
this works for me now.